sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -48,6 +48,13 @@ func resourceSysdigSecureCloudauthAccount() *schema.Resource {
 				Type:     schema.TypeBool,
 				Required: true,
 			},
+			"components": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem: &schema.Schema{
+					Type: v2.CloudauthAccountComponent,
```

Comment:
The `Type` key expects an int value from the enum defined in `valuetype.go` file.